Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:1.9.3-p551
+
+RUN apt-get update -qq && apt-get install -qy awscli curl && apt-get clean
+
+RUN useradd -c 'builder of ruby projs' -m -d /home/builder -s /bin/bash builder
+
+# RUN mkdir -p $HOME/.bundle
+# ENV BUNDLE_USER_HOME $HOME/.bundle
+# ENV BUNDLE_PATH $HOME/.bundle
+
+USER builder
+ENV HOME /home/builder
+RUN mkdir -p $HOME/app
+ADD --chown=builder:builder . $HOME/app/
+WORKDIR $HOME/app/
+
+RUN gem install bundler -v 2.0.1 && bundle install
```

Comment:
Change to 1.15.4

Reply:
Use the new docker file at https://github.com/dataxu/auth0-api/blob/0ed95a4ae0418178ea8bd5f56edc02be14562b95/Dockerfile